chrome/content/zotero/preferences/preferences_export.xul

```diff
@@ -49,10 +50,15 @@
 			<label value="&zotero.preferences.quickCopy.defaultOutputFormat;" control="quickCopy-menu"/>
 			<menulist id="zotero-quickCopy-menu"/>
 			
-			<separator class="thin"/>
-			
-			<checkbox id="zotero-quickCopy-copyAsHTML" label="&zotero.preferences.quickCopy.copyAsHTML;"
-				oncommand="Zotero_Preferences.Export.buildQuickCopyFormatDropDown(document.getElementById('zotero-quickCopy-menu'), this.checked ? 'html' : '');"/>
+			<hbox align="center">
+				<label value="&zotero.preferences.quickCopy.locale;" control="locale-menu"/>
+				<menulist id="locale-menu"/>
```

Comment:
But that calls a CSL editor-dedicated function that retrieves the value by id https://github.com/zotero/zotero/pull/623/files#diff-739aa5aa310d222a028ff24e416ff344R139 I'm not seeing why it's important to keep these IDs the same. I don't expect any of these elements to ever appear in the same window (at least not in current code), but having un-namespaced IDs, especially in large chunks of code like the preference window, is not a good idea. The IDs can be short and simple in small popup windows, like the site-specific quick-copy popup.